modules/reference/pages/systems/system-details/sd-overview.adoc

```diff
@@ -0,0 +1,179 @@
+Apply updates by navigating to menu:System Details[Packages].
+
+
+
+== System Info
+
+This section shows general information about the client.
+
+[[sdoverview-sysinfo-columns]]
+.System Info Columns
+[cols="1,1,1", options="header"]
+|===
+
+| Column Name
+| Description
+| Conditions
+
+| Hostname
+| The hostname used by the client with the {productname} Server.
+| -
+
+| FQDN
+| The fully qualified domain name of the client.
+| Shown only if it differs from the host name.
+//?? --LKB 2020-07-14
+
+| IP address
+| The IPv4 IP address of the client.
+| -
+
+| IPv6 address
+| The IPv6 IP address of the client.
+| -
+
+| Minion ID
+| The minion ID of the client.
+| Salt clients only.
+
+| Virtualization
+| The type of virtualization used by the client.
+| Virtualized clients only.
+
+| UUID
+| The universally unique identifier.
+| -
+
+| Kernel
+| The kernel currently in use on the client.
+| -
+
+| {productname} System ID
+| A unique identifier generated by {productname} when the client is registered.
+| -
+
+| Activation key
+| The activation key used to register the client.
+| Remains blank if the client has not been activated.
+
+| Installed products
+| Software products currently installed on the client.
+| -
+
+| Lock status
+| Indicates whether the client is locked.
+| Traditional clients only.
+Locked Salt clients instead show a yellow banner on this page.
+
+|===
+
+For more information about system locks, see xref:client-configuration:system-locking.adoc[].
+
+
+
+== Subscribed Channels
+
+This section shows the base and child software channels this client is subscribed to.
+
+Click btn:``Alter Channel Subscriptions`` to change the channel subscriptions for this client.
+Click a channel name to go to the [guimenu]``Basic Channel Details`` section.
+
+For more information about channels, see xref:client-configuration:channels.adoc[].
+
+
+
+== System Events
+
+This section shows the most recent system events for this client.
+
+[[sdoverview-sysevents-columns]]
+.System Events Columns
+[cols="1,1", options="header"]
+|===
+
+| Column Name
+| Description
+
+| Checked in
+| The time that the client last successfully checked in with the {productname} Server.
+
+| Registered
+| The time that the client registered with {productname}.
+
+| Last booted
+| The time that the client was most recently started or rebooted.
+
+|===
+
+
+Click btn:``Schedule system reboot`` to set a time for the client to reboot.
+The reboot will occur when the client next checks in with the {productname} Server after the scheduled time.
+For more on task scheduling, see xref:administration:task-schedules.adoc[].
+
+
+
+== System Properties
+
+This section shows further information about the client.
+
+[[sdoverview-sysproperties-columns]]
+.System Properties Columns
+[cols="1,1", options="header"]
+|===
+
+| Column Name
+| Description
+
+| System types
+| Lists the system types and add-on types currently applied to the client.
```

Comment:
I'm not about add-on types here.  I think this is just [salt] or [management] (for "tradiitional")

Reply:
I wasn't sure about that. Thanks!